project/src/const.js

```diff
@@ -17,12 +17,74 @@ const SortType = {
   TO_LOW: 'Price: high to low',
   TOP_RATE: 'Top rated first',
 };
-const OfferCity = {
+/* const OfferCity = {
   PARIS: 'Paris',
   COLOGNE: 'Cologne',
   BRUSSELS: 'Brussels',
   AMSTERDAM: 'Amsterdam',
   HAMBURG: 'Hamburg',
   DUSSELDORF: 'Dusseldorf',
+}; */
+const OfferCity = {
+  PARIS: {
+    name: 'Paris',
+    location: {
+      lat: 48.864716,
+      lng: 2.349014,
+      zoom: 12,
+    },
+  },
+  COLOGNE: {
+    name: 'Cologne',
+    location: {
+      lat: 50.935173,
+      lng: 6.953101,
+      zoom: 12,
+    },
+  },
+  BRUSSELS: {
+    name: 'Brussels',
+    location: {
+      lat: 50.85045,
+      lng: 4.34878,
+      zoom: 16,
+    },
+  },
+  AMSTERDAM: {
+    name: 'Amsterdam',
+    location: {
+      lat: 52.38333,
+      lng: 4.9,
+      zoom: 12,
+    },
+  },
+  HAMBURG: {
+    name: 'Hamburg',
+    location: {
+      lat: 53.551086,
+      lng: 9.993682,
+      zoom: 12,
+    },
+  },
+  DUSSELDORF: {
+    name: 'Dusseldorf',
+    location: {
+      lat: 51.233334,
+      lng: 6.783333,
+      zoom: 12,
+    },
+  },
+};
+const AuthorizationStatus = {
+  AUTH: 'AUTH',
+  NO_AUTH: 'NO_AUTH',
+  UNKNOWN: 'UNKNOWN',
+};
+const APIRoute = {
+  OFFERS: '/hotels',
+  LOGIN: '/login',
+  LOGOUT: '/logout',
+  //NEARBY: `/hotels/:${hotelIid}/nearby`,
+  NEARBY: `hotels/${1}/nearby`,
```

Comment:
```suggestion
  NEARBY: `/nearby`,
```
А дальше в проекте используй так `${APIRoute.OFFERS}/${offerId}${APIRoute.NEARBY}`